doc/build/dts/howtos.rst

```diff
@@ -120,23 +120,6 @@ device may not be initialized, or may have failed to initialize, you must verify
 that the device is ready to be used before passing it to any API functions.
 (This check is done for you by :c:func:`device_get_binding`.)
 
-In some situations the device cannot be known at build-time, e.g., if it depends
```

Comment:
keep this section, its still relevant.  Change `device_get_binding(DT_LABEL(MY_SERIAL))` to device_get_binding(DEVICE_DT_NAME(MY_SERIAL));`

Reply:
Added back.